vispy/visuals/image.py

```diff
@@ -157,10 +204,19 @@ class ImageVisual(Visual):
               entire view, with texture coordinates determined by the
               transform. This produces the best transformation results, but may
               be slow.
-
     grid: tuple (rows, cols)
         If method='subdivide', this tuple determines the number of rows and
         columns in the image grid.
+    polar: tuple (dir, loc, ori)
+        If given, a polar representation of the image will be created with:
```

Comment:
My inclination is to have the `polar` arg be a dict instead of a tuple. A bit more verbose, but also more clear. Plus you could apply defaults. But other devs should weigh in on this.

Reply:
Yes, that would be possible :+1:

Reply:
Done